src/test/java/com/fasterxml/jackson/databind/deser/std/EmptyStringAsSingleValueTest.java

```diff
@@ -0,0 +1,115 @@
+package com.fasterxml.jackson.databind.deser.std;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.cfg.CoercionAction;
+import com.fasterxml.jackson.databind.cfg.CoercionInputShape;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.List;
+
+public class EmptyStringAsSingleValueTest {
+    private static ObjectMapper normalMapper() {
+        ObjectMapper mapper = new ObjectMapper();
+        mapper.enable(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY);
```

Comment:
Could you change this to use builder pattern (`JsonMapper.builder().enable(...).build()`) instead? That way it'll merge to 3.0 cleanly.